packages/fxa-auth-server/lib/routes/attached-clients.js

```diff
@@ -258,17 +246,20 @@ module.exports = (log, db, devices, clientUtils) => {
             request,
             payload.deviceId
           );
+
           if (
             payload.sessionTokenId &&
-            destroyedDevice.sessionTokenId !== payload.sessionTokenId
+            destroyedDevice.sessionTokenId?.toString('hex') !==
```

Comment:
Instead of converting to hex here we ought to do it where we pull it out of the db, which is probably the source of this bug since the db layer changed most recently.

Here https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/db/models/auth/device.ts#L196

let's return something like this:
```js
return {
  sessionTokenId: uuidTransformer.from(result.sessionTokenId),
  refreshTokenId: uuidTransformer.from(result.refreshTokenId)
};
```

Reply:
Sounds good! I was tempted to put this in the DB layer, but was unsure if that was too aggressive for this fix. Thanks for the feedback.